CHANGELOG.md

```diff
@@ -67,6 +67,24 @@ url(r'^cms-api/', include('allink_core.core_apps.allink_cms.urls', namespace='cm
 ###### DATA MIGRATIONS
 
 ### NEW
+- added AllinkSEOAccordion plugin 
+    - the following entrys have to be added in allink_settings in order to use the plugin:
+    ```python
+    ALLINK_INSTALLED_APPS = [
+         ...
+        'allink_core.core_apps.allink_seo_accordion',
+    ]
+    CMS_ALLINK_CONTENT_PLUGIN_CHILD_CLASSES = [
+        ...
+        'CMSAllinkSEOAccordionContainerPlugin',
+    ]
+    ALLINK_LOCALE_PATHS = [
+        ...
+        '/app/allink_core/core_apps/allink_seo_accordion/locale',
+    ]
+    ...
+    ```
+    
```

Comment:
Das ist hier nicht nötig, da es ja im allink_core/core/allink_settings.py bereits gesetzt ist. Wenn man den allink_core in einem Projekt aktualisert, sollte das allink_seo_accordion direkt installiert werden und aktiv sein.